pulp_2to3_migrate/app/viewsets.py

```diff
@@ -0,0 +1,51 @@
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import mixins
+from rest_framework.decorators import detail_route
+
+from pulpcore.plugin.serializers import AsyncOperationResponseSerializer
+from pulpcore.plugin.tasking import enqueue_with_reservation
+from pulpcore.plugin.viewsets import (
+    NamedModelViewSet,
+    OperationPostponedResponse,
+)
+
+from .constants import PULP_2TO3_MIGRATION_RESOURCE
+from .models import MigrationPlan
+from .serializers import (
+    MigrationPlanSerializer,
+    MigrationPlanRunSerializer,
+)
+from .tasks.migrate import migrate_from_pulp2
+
+
+class MigrationPlanViewSet(NamedModelViewSet,
+                           mixins.CreateModelMixin,
+                           mixins.RetrieveModelMixin,
+                           mixins.DestroyModelMixin,
+                           mixins.ListModelMixin):
+    endpoint_name = 'migration-plans'
+    queryset = MigrationPlan.objects.all()
+    serializer_class = MigrationPlanSerializer
+
+    @swagger_auto_schema(
```

Comment:
please add 'operation_summary' to this swagger_auto_schema. Set  it to "Run migration plan".

Reply:
done